src/Net/LinkShortener.php

```diff
@@ -29,7 +30,7 @@ public static function Get(string $shortLink): ?string {
 
     public static function Create(string $longUrl): string {
         $redis = self::GetRedisInstance();
-        while($key = self::GenerateShortString()) {
+        while($key = self::GenerateShortString($longUrl)) {
             if ($redis->get($key) === false) {
```

Comment:
Same idea here. This was meant to check if the key wasn't already in the redis db, but I don't care if the key exists anymore since duplicates are expected, so don't bother with this check.